sdk/core/Azure.Core/src/Pipeline/Internal/HttpEnvironmentProxy.cs

```diff
@@ -210,7 +210,7 @@ private bool IsMatchInBypassList(Uri input)
                 {
                     if (s[0] == '.')
                     {
-                        // This should match either domain it self or any subdomain or host
+                        // This should match either domain it self or any sub-domain or host
```

Comment:
I think subdomain is correct:
https://en.wikipedia.org/wiki/Subdomain

Reply:
```suggestion
                        // This should match either domain it self or any subdomain or host
```